compiler/src/dotty/tools/dotc/cc/Setup.scala

```diff
@@ -28,10 +29,9 @@ trait SetupAPI:
 
   /** Setup procedure to run for each compilation unit
    *   @param tree       the typed tree of the unit to check
-   *   @param recheckDef the recheck method to run on completion of symbols with
-   *                     inferred (result-) types
+   *   @param checker    the capture checker which will run subsequently.
    */
-  def setupUnit(tree: Tree, recheckDef: DefRecheck)(using Context): Unit
```

Comment:
Could `type DefRecheck` be removed? It seemed it's used anymore after this change.